web/src/assets/schema/base.json

```diff
@@ -108,92 +127,52 @@
         }
       ]
     },
-    "contact": {
-      "$id": "#/properties/contact",
-      "type": "object",
-      "title": "Contact",
-      "required": [
-        "email",
-        "name",
-        "affiliations"
-      ],
-      "properties": {
-        "email": {
-          "$id": "#/properties/contact/properties/email",
-          "type": "string",
-          "title": "Email"
-        },
-        "name": {
-          "$id": "#/properties/contact/properties/name",
-          "type": "string",
-          "title": "Name"
-        },
-        "affiliations": {
-          "$id": "#/properties/contact/properties/affiliations",
-          "type": "string",
-          "title": "Affiliations"
-        }
-      }
-    },
     "sponsors": {
       "$id": "#/properties/sponsors",
       "type": "object",
```

Comment:
sponsors should be an array.

also we should a string field to the object called `Award number`

Reply:
> sponsors should be an array.

Oops, forgot to actually update the type itself. 

> also we should a string field to the object called `Award number`

To the sponsors object (the objects contained within the sponsors array)?

Reply:
yes, another field to the sponsors object:

```
- sponsors:
   - name:
     ...
     award_number:
```